Probe/src/data_collector.py

```diff
@@ -15,25 +17,38 @@ def dataCapture(noOfPackets = 2,filename = "f"):
         * none 
     '''
 
-    path = "../capture"
-    if not os.path.exists(path):   
-        os.makedirs(path)
-
-    path_packet = f"../capture/{filename}.pcap"
-
-    capture = scapy.sniff(filter="ip6", count = noOfPackets)
-    scapy.wrpcap(path_packet,capture)
-    print(f"captured {noOfPackets} packets")
+    path_capture = os.getenv('CAPTURED_PACKET_PATH')
+    if not os.path.exists(path_capture):  
+        try: 
+            os.makedirs(path_capture)
+        except(e):
```

Comment:
Do not do wild card exception handling, try to catch the specific exception and use a system exit and logging to trace the error

Reply:
I've added some specific exception, tried logging and sys exit